app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::OrganizationsController < Api::ApiController
+  include FilterByOwner
+  include FilterByCurrentUserRoles
+  # include TranslatableResource
+  include IndexSearch
+  include AdminAllowed
+
+  require_authentication :update, :create, :destroy, scopes: [:organization]
+
+  resource_actions :show, :index, :create, :update, :deactivate
+  schema_type :json_schema
+
+  prepend_before_action :require_login,
+    only: [:create, :update, :destroy]
+
+  def create
+    organizations = Organization.transaction do
+      Array.wrap(params[:organizations]).map do |organization_params|
+        Organizations::Create.with(api_user: api_user).run!(organization_params)
+      end
+    end
+
+    created_resource_response(organizations)
+  end
+
+  def update
+    Organization.transaction do
+      Array.wrap(resource_ids).zip(Array.wrap(params[:organizations])).map do |organization_id, organization_params|
```

Comment:
Pretty sure our v1 routes require an id for update / delete operations, e.g. `/organisations/1`. I think ths means that `resource_ids == resource_id` and so we can probably clean up the outer loop on this.